user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps.md

```diff
@@ -0,0 +1,103 @@
+
+This component is used to display markers and/or lines on a map. It uses one or more GQI queries as data input.
+
+> [!NOTE]
+> To use the Maps component, the host servers for DataMiner Maps have to be configured in the file *C:\Skyline DataMiner\Maps\ServerConfig.xml*. If this file does not exist, it will be created automatically when you use a Maps component for the first time. To change the configuration, see [Configuring the DataMiner Maps host servers](xref:Configuring_the_DataMiner_Maps_host_servers).
+
+## Adding data
+
+Add one or multiple GQI data sources to the component. See [Applying a data feed](xref:Apply_Data_Feed).
+
+The data is visualized as a set of markers by default, but it can also be configured as a set of lines. In both cases, the component will try to automatically apply the dimensions needed to show the visualization of the data.
+
+For markers, the process of applying the dimensions works as follows:
+
+1. As a marker needs an ID, a latitude, and a longitude value to be visualized, the component will look for columns named "ID", "Latitude"/"Lat", and "Longitude"/"Lng" in your query result.
+
+1. If these columns exist, a marker is configured for each query row in the result, taking the values of the ID, latitude, and longitude columns for that row.
+
+1. If no match is found, the component will take the first column containing string values for the ID and the first two columns containing numbers for latitude and longitude while taking into account the columns that are already assigned to other dimensions.
+
+1. The chosen column values are displayed for each query. If a dimension is set incorrectly, you can still change it manually at any point.
+
+This process is similar for the configuration of lines, but instead of one latitude and longitude value, two pairs of coordinates need to be configured, one for the starting point and one for the end point of the line.
+
+## Layer settings
+
+On the *Component > Layout* tab, you can configure the following settings under *Layer settings*:
+
+- *Weight*: Determines which marker is put in the foreground and which in the background in case markers or lines from different layers overlap with each other. The higher the weight of the layer, the more it will be pushed to the foreground. The default weight is 1.
+
+- *Query type*: By default, each query is configured as a set of markers, which is indicated with a circle icon next to the name of the query. If you hover the mouse pointer over the query name, a line icon will also become visible. Click this icon to change the visualization of the data from a set of markers to a set of lines and to reconfigure the dimensions so they can be visualized as lines.
+
+- *Dimensions*: For each dimension, a dropdown is shown where the chosen column for that dimension can be changed. All dropdowns contain only valid options for that dimension, i.e. string valued columns for the ID and numeric valued columns for the latitude and longitude.
```

Comment:
Query type / Dimensions: the way these two settings are currently described creates the impression that there is actually a "Query type" and "Dimensions" option in the UI, but as far as I can tell, this is not the case. We'd best adjust this to avoid confusion.

Reply:
@MariekeGO There is a setting to change the query type (the circle and line buttons), and there is a dropdown setting for each dimension needed to configure the lines or markers, only these settings aren't labeled. What would be the best way to explain these settings?